report.md

```diff
@@ -867,7 +887,73 @@ This is a best-practice to protect against reentrancy in other modifiers.
 
 
 
-## NC-7: Large literal values multiples of 10000 can be replaced with scientific notation
+## NC-7: Empty Block
+
+Consider removing empty blocks.
+
+- Found in src/AdminContract.sol [Line: 14](tests/contract-playground/src/AdminContract.sol#L14)
+
+	```solidity
+	    function someOtherImportantThing() external nonReentrant onlyOwner {
+	```
+
+- Found in src/EmptyBlocks.sol [Line: 22](tests/contract-playground/src/EmptyBlocks.sol#L22)
+
+	```solidity
+	    constructor() {
```

Comment:
I think this should capture the nested empty block on line 24, instead of the constructor definition. What do you think?

```sol
    constructor() {
        emit EmptyEvent();
        // Empty Block Below
        {

        }
    }
```

Reply:
Easy. I'll do it right away !

Reply:
For now I'm capturing Line 25 which where the `{` starts.